src/components/EpigraphyView.vue

```diff
@@ -47,8 +62,10 @@ export default {
 
   methods: {
     async getEpigraphyInfo() {
+      this.loading = true
       let result = await axios.get('https://oare-test.herokuapp.com/api/textEpigraphies/'
```

Comment:
We just need to eventually rewrite these requests to go through the vuex store correct?

Reply:
Not necessarily. At this point the app is simple enough that Vuex would probably be a little much, but that's not saying we could get there eventually.